cg/services/orders/store_order_services/store_fastq_order_service.py

```diff
@@ -81,56 +55,49 @@ def create_maf_case(self, sample_obj: Sample, order: Order) -> None:
         order.cases.append(case)
         self.status_db.session.add_all([case, relationship])
 
-    def store_items_in_status(
-        self, customer_id: str, order: str, ordered: datetime, ticket_id: str, items: list[dict]
-    ) -> list[Sample]:
+    def store_items_in_status(self, order: FastqOrder) -> list[Sample]:
         """Store fastq samples in the status database including family connection and delivery"""
+        ticket_id: str | None = order.ticket_number
         customer: Customer = self.status_db.get_customer_by_internal_id(
-            customer_internal_id=customer_id
+            customer_internal_id=order.customer
         )
-        if not customer:
-            raise OrderError(f"Unknown customer: {customer_id}")
         new_samples = []
         case: Case = self.status_db.get_case_by_name_and_customer(
             customer=customer, case_name=ticket_id
         )
-        submitted_case: dict = items[0]
         status_db_order = Order(
             customer=customer,
             order_date=datetime.now(),
             ticket_id=int(ticket_id),
         )
         with self.status_db.session.no_autoflush:
-            for sample in items:
+            for sample in order.samples:
                 new_sample = self.status_db.add_sample(
-                    name=sample["name"],
-                    sex=sample["sex"] or "unknown",
-                    comment=sample["comment"],
-                    internal_id=sample.get("internal_id"),
-                    order=order,
-                    ordered=ordered,
+                    name=sample.name,
+                    sex=sample.sex or "unknown",
+                    comment=sample.comment,
+                    internal_id=sample._generated_lims_id,
+                    order=order.name,
+                    ordered=datetime.now(),
```

Comment:
What was the "ordered" variable we're replacing with datetime.now?

Reply:
Good catch! Seems to have been the "date" returned when submitting the data to LIMS....

![image](https://github.com/user-attachments/assets/acdecad8-1080-43ad-8381-ee954c22bd38)

![image](https://github.com/user-attachments/assets/16df0e03-4718-416e-bd9e-c1c8b03487bd)

Would you prefer we keep it that way?

Reply:
hahaha, this logic 🔫 Let's use datatime.now()